cuml/kmeans_demo.ipynb

```diff
@@ -0,0 +1,271 @@
+    "\n",
+    "cuML’s KMeans supports the fast KMeans++ intialization method. This method is more stable than randomnly selecting K points.\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "|  Parameters: |  Description:    |\n",
+    "| ------------ | ---------------- |\n",
+    "| `n_clusters:int (default = 8)` | The number of centroids or clusters you want. |\n",
+    "| `max_iter:int (default = 300)` | The more iterations of EM, the more accurate, but slower. |\n",
+    "| `tol:float (default = 1e-4)` | Stopping criterion when centroid means do not change much. |\n",
+    "| `verbose:boolean (default = 0)` | If True, prints diagnositc information. |\n",
+    "| `random_state:int (default = 1)` | If you want results to be the same when you restart Python, select a state. |\n",
+    "| `precompute_distances:boolean (default = ‘auto’)` | Not supported yet. |\n",
+    "| `init: (default = 'scalable-k-means++')` | 'scalable-k-means++' or 'k-means\\|\\|': Uses fast and stable scalable kmeans++ intialization. <br> 'random' or an ndarray: Choose 'n_cluster' observations (rows) at random from data for the initial centroids. <br> If an ndarray is passed, it should be of shape (n_clusters, n_features) and gives the initial centers. |\n",
+    "| `n_init:int (default = 1)` | Number of times intialization is run. More is slower, but can be better. |\n",
+    "| `algorithm:“auto”` | Currently uses full EM, but will support others later. |\n",
+    "| `n_gpu:int (default = 1)` | Number of GPUs to use. If -1 is used, uses all GPUs. More usage is faster. |\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "|  Methods:         |  Description:      |\n",
+    "| ------------------| -----------------  |\n",
+    "|`fit(self, X)` | Compute k-means clustering with X. |\n",
+    "|`fit_predict(self, X)` | Compute cluster centers and predict cluster index for each sample. |\n",
+    "|`fit_transform(self, input_gdf)` | Compute clustering and transform input_gdf to cluster-distance space. |\n",
+    "|`get_params(self[, deep])` | Sklearn style return parameter state |\n",
+    "|`predict(self, X)` | Predict the closest cluster each sample in X belongs to. |\n",
+    "|`set_params(self, **params)` | Sklearn style set parameter state to dictionary of params. |\n",
+    "|`transform(self, X)` | Transform X to a cluster-distance space. |\n",
+    "\n",
+    "<br>\n",
+    "    \n",
+    "The model accepts only numpy arrays or cudf dataframes as the input. In order to convert your dataset to cudf format please read the cudf documentation on https://rapidsai.github.io/projects/cudf/en/latest/. For additional information on the DBSCAN model please refer to the documentation on https://rapidsai.github.io/projects/cuml/en/latest/index.html\n",
```

Comment:
Please replace `The model accepts only numpy arrays or cudf dataframes as the input` with `The model can take array-like objects, either in host as NumPy arrays or in device (as Numba or _cuda_array_interface_compliant), as well as cuDF DataFrames as the input.`